std_msgs/UInt32[] id_array
geometry_msgs/Pose[] pose_array
